src/python/aqueduct_executor/operators/connectors/tabular/spec.py

```diff
@@ -44,10 +44,12 @@ def unwrap_connector_config(cls, connector_config, values):  # type: ignore
     if "connector_name" not in values:
         raise ValueError("Unknown connector name.")
 
-    values["connector_name"]
-
     if not isinstance(connector_config, dict):
         raise ValueError("connector_config is not a dictionary.")
+    
+    if "conf" not in connector_config:
```

Comment:
This is necessary for Airflow parsing of the spec

Reply:
Can you add a comment explaining why this is custom logic for Airflow?